app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,10 @@
+class RecipesController < ApplicationController
+  def show
+    @recipe = Recipe.find(params[:id])
+  end
+
+  private
+    def recipe_params
+      params.permit(:name, :complexity, :genre)
```

Comment:
Good job creating this strong params method. Again, be careful of what you build and when; only build methods or routes (or anything really) as you need it according to the user stories.